RELEASE.md

```diff
@@ -0,0 +1,30 @@
+# Release Process
+- [ ] Pull latest from `main` branch.
+- [ ] Determine release version string dependending on whether changes included
+      in changelog are API breaking, it's a release candidate, etc.
+      The release string should have the format "vX.Y.Z", with a possible
+      trailing "-rcN" and follow [semantic versioning](https://semver.org/).
+- [ ] Checkout branch named `release-N` where N is the version string.
+- [ ] Update the version field in the package entry of `Cargo.toml` files.
+  - The versions must be the same.
+  - There are 4 relevant `Cargo.toml` files in the repo:
+    - `Cargo.toml`: update the version string.
+    - `cairo-vm-cli/Cargo.toml`: update the version string and also the `cairo-vm` dependency version to match the above.
+    - `felt/Cargo.toml`: update the version string.
+    - `deps/parse-hyperlinks/Cargo.toml`: this vendored dependency needs its version bumped, but does not need to match the other crate versions.
```

Comment:
I think it's better to just split this, as this becomes busywork. I'll try to create a new repo for it this week (as "when I have time to send a patch to nom" seems not to happen) so we can update it separately and not need bumping it every time.